src/applications/burials-v2/containers/ConfirmationPage.jsx

```diff
@@ -1,137 +1,190 @@
-                  .format('MMM D, YYYY h:mm a [CT]')}
-              </span>
-            </li>
-            <li>
-              <h4>Deceased Veteran</h4>
-              <span>
-                {veteranName.first} {veteranName.middle} {veteranName.last}{' '}
-                {veteranName.suffix}
-              </span>
-            </li>
-            <li>
-              <h4>Benefits claimed</h4>
-              <ul className="benefits-claimed">
-                {Object.entries(benefits).map(([benefitName, isRequested]) => {
-                  const label = benefitsLabels[benefitName];
-                  return isRequested && label ? (
-                    <li key={benefitName}>{label}</li>
-                  ) : null;
-                })}
-              </ul>
-            </li>
-            {hasDocuments && (
-              <li>
-                <h4>Documents uploaded</h4>
-                {deathCertificate && <p>Death certificate: 1 file</p>}
-                {transportationReceipts && (
-                  <p>
-                    Transportation documentation:{' '}
-                    {transportationReceipts.length}{' '}
-                    {transportationReceipts.length > 1 ? 'files' : 'file'}
-                  </p>
-                )}
-              </li>
-            )}
-            <li>
-              <h4>Your claim was sent to</h4>
-              <address className="schemaform-address-view">
-                {response?.regionalOffice?.map((line, index) => (
-                  <p key={index}>{line}</p>
-                ))}
-              </address>
-            </li>
-          </ul>
-          <va-button
-            className="usa-button screen-only"
-            onClick={() => window.print()}
-            text="Print for your records"
-          />
-        </div>
-        <div className="confirmation-guidance-container">
-          <h3 className="confirmation-guidance-heading">Need help?</h3>
-          <p className="confirmation-guidance-message">
-            If you have questions, call <va-telephone contact="8008271000" />,
-            Monday through Friday, 8:00 a.m. to 9:00 p.m. ET. Please have your
-            Social Security number or VA file number ready. For
-            Telecommunication Relay Services, dial{' '}
-            <va-telephone contact="711" />.
+          )}
+          <li>
+            <h4>Your application was sent to</h4>
+            <address className="schemaform-address-view">
+              {response?.regionalOffice?.map((line, index) => (
+                <p key={index}>{line}</p>
+              ))}
+            </address>
+          </li>
+          <li>
+            <h4>Confirmation for your records</h4>
+            <p>You can print this confirmation page for your records</p>
+          </li>
+        </ul>
+        <va-button
+          className="usa-button screen-only"
+          onClick={() => window.print()}
+          text="Print this page"
+          uswds="false"
```

Comment:
reason for `uswds="false"`?

Reply:
@powellkerry same as the above - will remove

Reply:
we will not launch with uswds:false/true
I have created Issue [#79820 ](https://app.zenhub.com/workspaces/benefits-non-disability-645913c7d909c20011380ae8/issues/gh/department-of-veterans-affairs/va.gov-team/79820) to clean up this request.

We will not be making this change as part of this PR.